### PR TITLE
Cr 13510

### PR DESCRIPTION
**Pull request by eti-codefresh**

### CODE REVIEW

File: workflows/codefresh-csdp/versions/1.1.3/images/report-image-info/src/registry-client.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    throw new Error('Registry credentials is required parameter. Add one from following registry parameters in your workflow to continue:\n - Docker credentials: DOCKERHUB_USERNAME, DOCKERHUB_PASSWORD\n - GCR credentials: GCR_KEY_FILE_PATH\n - AWS registry credentials: AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGION\n - Standard registry credentials: REGISTRY_USERNAME, REGISTRY_PASSWORD, REGISTRY_DOMAIN');
+}
+
+module.exports = {
+    parseImageName,
+    async getRegistryClient(image) {
+    if (inputs.retrieveCredentialsByDomain) {
+        return createRegistryClientByImage(image);
+    }
+    return createRegistryClient(image);
```
Comment (yaroslav-codefresh) [inline workflows/codefresh-csdp/versions/1.1.3/images/report-image-info/src/registry-client.js:165]: please extract the function and bind if you need `this` instance or just export if not

File: workflows/codefresh-csdp/versions/1.1.3/rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-hub.codefresh-csdp.1.1.3
```
Comment (yaroslav-codefresh) [inline workflows/codefresh-csdp/versions/1.1.3/rbac.yaml:4]: maybe @denis-codefresh is the correct person to address -- but why do we need to create a new service account each time with different version?

File: workflows/codefresh-csdp/versions/1.1.3/images/report-image-info/src/registry-client.js (hunk 1 above)
Comment (eti-codefresh) [reply workflows/codefresh-csdp/versions/1.1.3/images/report-image-info/src/registry-client.js]: done

File: workflows/codefresh-csdp/versions/1.1.3/docs/report-image-info.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* REGISTRY_PASSWORD (required) - The Kubernetes secret with the standard registry password
+* PASSWORD_SECRET_KEY (optional) The key in the Kubernetes secret with the standard registry password. Default is 'password'
+* REGISTRY_DOMAIN (required) - The Kubernetes secret with the standard registry domain
+* REGISTRY_DOMAIN_SECRET_KEY (optional) - The key in the Kubernetes secret with the standard registry domain. Default is 'domain'
+* AWS_ROLE_SECRET (required) - The Kubernetes secret with the Amazon role
+* AWS_ROLE_SECRET_KEY (optional) -The key in the Kubernetes secret with the standard Amazon role. Default is 'role'
+
+### Outputs
+* `image-name` – name of the reported image
+* `image-sha` – SHA of the reported image
```
Comment (denis-codefresh) [inline workflows/codefresh-csdp/versions/1.1.3/docs/report-image-info.md:45]: add new output here

**Review by denis-codefresh — APPROVED** [review]

**Pull request merged**